### ISSUE TITLE
[core] Project copyright & description are now required

### ISSUE THREAD

**Issue by aalmiray** [jreleaser/jreleaser#918]
### Task List

- [x] Steps to reproduce provided
- [ ] Stacktrace (if present) provided
- [ ] Example that reproduces the problem (link to git repository is ideal)
- [x] Full description of the issue provided (see below)

### Steps to Reproduce

1. Configure a project to be used with the CLI.
2. Follow the [simple release](https://jreleaser.org/guide/latest/examples/simple-release.html) example.
3. Perform a release.

### Expected Behaviour

A successful release posted.

### Actual Behaviour

Error because both project copyright and description are missing

```
[ERROR] == JReleaser ==
[ERROR] project.copyright must not be blank
[ERROR] project.description must not be blank
JReleaser has not been properly configured.
project.copyright must not be blank
project.description must not be blank
```

### Environment Information

- **Operating System**: N/A
- **JReleaser Version:** 1.2.0
- **JDK Version:** N/A

**Comment by aalmiray** [comment]
cc: @sormuras

**Comment by aalmiray** [comment]
Released in v1.3.0 -> https://github.com/jreleaser/jreleaser/releases/tag/v1.3.0
